### PR TITLE
[TASK] Deprecate INCLUDE_TYPOSCRIPT syntax

### PR DESCRIPTION
**Pull request by linawolf**
Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1070

Releases: main

### CODE REVIEW

File: Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst [hunk 1]
```diff
@@ -6,45 +6,25 @@
 File imports
 ============
 
+..  deprecated:: 13.4
+    The old school :typoscript:`<INCLUDE_TYPOSCRIPT:` syntax has been deprecated
+    and will be removed with TYPO3 14.0. See :ref:`typoscript-syntax-includes-migration`.
```
Comment (brotkrueml) [inline Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst:11]:
```suggestion
    and will be removed with TYPO3 v14.0. See :ref:`typoscript-syntax-includes-migration`.
```

File: Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst [hunk 2]
```diff
@@ -6,45 +6,25 @@
...
-        page {
-          # This import won't work!
-          @import 'EXT:my_extension/Configuration/TypoScript/bar.typoscript'
-          20 = TEXT
-          20.value = bar
-        }
+The keyword :typoscript:`@import` is a syntax construct and
+thus available in both frontend TypoScript and backend TSconfig.
+
+:typoscript:`@import` allows including additional files using wildcards the file
```
Comment (brotkrueml) [inline Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst:19]:
```suggestion
:typoscript:`@import` allows including additional files using wildcards on the file
```

File: Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst [hunk 3]
```diff
@@ -132,77 +112,123 @@ Some examples:
...
+    -<INCLUDE_TYPOSCRIPT: source="DIR:EXT:my_extension/Configuration/TypoScript/" extensions="typoscript">
+    +@import 'EXT:my_extension/Configuration/TypoScript/*.typoscript'
 
-  Paths relative to the including file can be used, if the inclusion is called
-  from inside a file. These paths start with :file:`FILE:./`. This allows simple, nested
-  TypoScript templates that can be moved or copied without the need to adapt all includes.
+    # Including .typoscript and .ts files in a single (non recursive!) directory
+    -<INCLUDE_TYPOSCRIPT: source="DIR:EXT:my_extension/Configuration/TypoScript/" extensions="typoscript,ts">
+    +@import 'EXT:my_extension/Configuration/TypoScript/*.typoscript'
+     # Rename all files ending on .ts to .typoscript
```
Comment (brotkrueml) [inline Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst:164]:
```suggestion
     # Rename all files ending from .ts to .typoscript
```

File: Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst [hunk 4]
```diff
@@ -132,77 +112,123 @@ Some examples:
...
+:typoscript:`@import` is a bit more restrictive.
 
-  Can be used instead of :typoscript:`FILE` to include multiple files at once.
-  It includes all files from a directory relative to :php:`\TYPO3\CMS\Core\Core\Environment::getPublicPath()`,
-  including subdirectories.
+As one restriction :typoscript:`@import` cannot include files from arbitrary directories
+like :file:`fileadmin/`, but only from extensions by using the :typoscript:`EXT:`
+prefix. Instances that use :typoscript:`<INCLUDE_TYPOSCRIPT:` with :typoscript:`source="FILE:./someDirectory/..."`
+should move this TypoScript into a project or site extension. Such instances are also encouraged to
+look into the TYPO3 v13 "Site sets" feature and eventually transition towards it along the way.
```
Comment (brotkrueml) [inline Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst:179]: I suggest to link "site sets" for easy access of the reader.

File: Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst [hunk 5]
```diff
@@ -132,77 +112,123 @@ Some examples:
...
 
-  Files are included in alphabetical. Also files are included first, then directories.
+:typoscript:`@import` allows to import files with the file ending `.typoscript`
```
Comment (brotkrueml) [inline Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst:181]: You also mentioned that above in an important section.

File: Documentation/Security/GuidelinesIntegrators/Typoscript.rst [hunk 6]
```diff
@@ -119,51 +119,26 @@ using `TypoScript`.
...
-.. code-block:: apacheconf
-   :caption: .htaccess
+External file inclusion
+=======================
 
-   <FilesMatch "\.typoscript">
-     deny from all
-   </FilesMatch>
+..  deprecated:: 13.4
+    The `<INCLUDE_TYPOSCRIPT` Syntax used for inclusion of TypoScript files
```
Comment (brotkrueml) [inline Documentation/Security/GuidelinesIntegrators/Typoscript.rst:129]:
```suggestion
    The `<INCLUDE_TYPOSCRIPT` syntax used for inclusion of TypoScript files
```

File: Documentation/Security/GuidelinesIntegrators/Typoscript.rst [hunk 7]
```diff
@@ -119,51 +119,26 @@ using `TypoScript`.
...
+    from the "fileadmin" or other publicly available folders has been deprecated
+    and will be removed with TYPO3 14.0. See migration.
```
Comment (brotkrueml) [inline Documentation/Security/GuidelinesIntegrators/Typoscript.rst:131]:
```suggestion
    and will be removed with TYPO3 v14.0. See migration.
```

And please link "migration".

File: Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst (hunk 5 above)
Comment (linawolf) [reply Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst:181]: Repetition cannot harm

File: Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst [hunk 8]
```diff
@@ -132,77 +112,123 @@ Some examples:
...
+:typoscript:`@import` is a bit more restrictive.
 
-  Can be used instead of :typoscript:`FILE` to include multiple files at once.
-  It includes all files from a directory relative to :php:`\TYPO3\CMS\Core\Core\Environment::getPublicPath()`,
-  including subdirectories.
+As one restriction :typoscript:`@import` cannot include files from arbitrary directories
+like :file:`fileadmin/`, but only from extensions by using the :typoscript:`EXT:`
+prefix. Instances that use :typoscript:`<INCLUDE_TYPOSCRIPT:` with :typoscript:`source="FILE:./someDirectory/..."`
+should move this TypoScript into a project or site extension. Such instances are also encouraged to
+look into the TYPO3 v13 "Site sets" feature and eventually transition towards it along the way.
```
Comment (linawolf) [inline Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst:179]:
```suggestion
look into the TYPO3 v13 :ref:`Site sets <site-sets>` feature and eventually transition towards it along the way.
```

File: Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst (hunk 4 above)
Comment (linawolf) [reply Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst]: done

File: Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst [hunk 9]
```diff
@@ -132,77 +112,123 @@ Some examples:
...
+There are a few more use cases that cannot be transitioned so easily since
+:typoscript:`@import` is a bit more restrictive.
 
-  Can be used instead of :typoscript:`FILE` to include multiple files at once.
-  It includes all files from a directory relative to :php:`\TYPO3\CMS\Core\Core\Environment::getPublicPath()`,
-  including subdirectories.
+As one restriction :typoscript:`@import` cannot include files from arbitrary directories
+like :file:`fileadmin/`, but only from extensions by using the :typoscript:`EXT:`
+prefix. Instances that use :typoscript:`<INCLUDE_TYPOSCRIPT:` with :typoscript:`source="FILE:./someDirectory/..."`
+should move this TypoScript into a project or site extension. Such instances are also encouraged to
```
Comment (brotkrueml) [inline Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst:178]:
```suggestion
should move these TypoScript files into a project or site extension. Such instances are also encouraged to
```

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**